testcases/unit/unit.mk

```diff
@@ -16,6 +16,8 @@ testcases_unit_policytest_CFLAGS=-I${top_srcdir}/usr/lib/common		\
 	-DSTDLL_NAME=\"policytest\" -I${top_srcdir}/usr/lib/config	\
 	-I${top_builddir}/usr/lib/config -I${top_builddir}/usr/lib/api
 
+testcases_unit_policytest_LDFLAGS=-lpthread
```

Comment:
Does policytest really need pthread ? I don't see that being used in policytest ....

Reply:
It seems to be needed explicitly on AIX, but is included implicitly on Linux (?). Without this, linking fails with unresolved pthread symbols.

Reply:
Hm on my system the executable does not need pthreads:
```
# ldd testcases/policy/policytest
        linux-vdso64.so.1 (0x000003ffea5fe000)
        libc.so.6 => /usr/lib64/libc.so.6 (0x000003ff9eb80000)
        /lib/ld64.so.1 (0x000003ff9ed80000)
```